src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -104,6 +104,11 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
     cancelPipelineRun({ name, namespace });
   };
 
+  delete = pipelineRun => {
```

Comment:
Would have expected the linter to complain about this since `delete` is a reserved keyword and could be problematic in some scenarios. Probably best to rename this to avoid future headaches.